common.gradle

```diff
@@ -0,0 +1,26 @@
+ext {
```

Comment:
It would be good to have words here about what constitutes common and is safe to put here.